src/cayenne/formats/citeproc.clj

```diff
@@ -274,9 +277,11 @@
         {:clinical-trial-number ctn :registry registry}
         (when (and type (not= type "-")) {:type type}))) ctns registries types)))
 
-(defn ->content-domains [solr-doc]
-  {:domain (or (get solr-doc "domains") [])
-   :crossmark-restriction (or (get solr-doc "domain_exclusive") false)})
+(defn ->content-domains [{:keys [crossmark-unaware?]} solr-doc]
+  (merge 
+    {:domain (or (get solr-doc "domains") [])}
+    (if-not crossmark-unaware?
```

Comment:
Did we say we'd remove the 'crossmark-unaware' idea for peer review in the end?

Reply:
I ended up leaving that because of the comment made about being able to disable crossmark for other content types in the future, and it also means for peer reviews we don't get an empty `crossmark-restrictions` node.

Reply:
Ok. Think this is the right decision for now.

Reply:
Actually this whole approach with the type dictionary is a neat solution.

Reply:
While it looks a bit awkward I went with `crossmark-unaware?` instead of `crossmark-aware?` because at this stage it made more sense for aware to be the default.

Reply:
Yeah agreed.